packages/snowpack/src/commands/dev.ts

```diff
@@ -272,28 +261,21 @@ export async function command(commandOptions: CommandOptions) {
     try {
       credentials = await readCredentials(cwd);
     } catch (e) {
-      console.error(
-        colors.red(
-          `✘ No HTTPS credentials found! Missing Files:  ${colors.bold(
-            'snowpack.crt',
-          )}, ${colors.bold('snowpack.key')}`,
-        ),
+      logger.error(
+        `✘ No HTTPS credentials found! Missing Files:  ${colors.bold(
+          'snowpack.crt',
+        )}, ${colors.bold('snowpack.key')}`,
       );
-      console.log();
-      console.log('You can automatically generate credentials for your project via either:');
-      console.log();
-      console.log(
-        `  - ${colors.cyan('devcert')}: ${colors.yellow('npx devcert-cli generate localhost')}`,
-      );
-      console.log('    https://github.com/davewasmer/devcert-cli (no install required)');
-      console.log();
-      console.log(
-        `  - ${colors.cyan('mkcert')}: ${colors.yellow(
-          'mkcert -install && mkcert -key-file snowpack.key -cert-file snowpack.crt localhost',
-        )}`,
-      );
-      console.log('    https://github.com/FiloSottile/mkcert (install required)');
-      console.log();
+      logger.info(`You can automatically generate credentials for your project via either:
+
+  - ${colors.cyan('devcert')}: ${colors.yellow('npx devcert-cli generate localhost')}
+    https://github.com/davewasmer/devcert-cli (no install required)
+
+  - ${colors.cyan('mkcert')}: ${colors.yellow(
+        'mkcert -install && mkcert -key-file snowpack.key -cert-file snowpack.crt localhost',
+      )}
+
+    https://github.com/FiloSottile/mkcert (install required)`);
```

Comment:
Minor note: since in dev we truncate logs, I changed this to be a single multiline message